notifier/notifier.go

```diff
@@ -383,25 +391,25 @@ func (n *Notifier) Stop() {
 
 // Describe implements prometheus.Collector.
 func (n *Notifier) Describe(ch chan<- *prometheus.Desc) {
```

Comment:
I'm for global by default, and only changing it when a user comes along that needs it split. They usually don't and it's a premature optimisation (and increase in complexity) to do it earlier. Even for libraries with multiple instances, many metrics can stay global indefinitely (e.g. Julius's case) and making things more complex results in less instrumentation happening.